src/server.py

```diff
@@ -437,25 +405,13 @@ def run_seta_query():
 @app.route("/data/setadetails/")
 @json_response
 def run_seta_details_query():
-    # TODO: remove inactive when buildbot api queries s/inactive/priority/
-    inactive = sanitize_bool(request.args.get("inactive", 0))
```

Comment:
About this, after we remove inactive variable, we should change the way which buildbot make request. The buildbot should use priority instead of inactive.